packages/klevu-ui/src/components/klevu-chat-messages/klevu-chat-messages.tsx

```diff
@@ -214,15 +214,21 @@ export class KlevuChatMessages {
             return (
               <div>
                 <klevu-slides
+                  center-next-prev={true}
```

Comment:
we need to provide prop as centerNextPrev?
also if providing true, then no need to pass value, just pass prop